src/app/app.component.spec.ts

```diff
@@ -16,16 +23,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'angular-app'`, () => {
+  it(`should have as title 'AngularApp'`, () => {
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.debugElement.componentInstance;
-    expect(app.title).toEqual('angular-app');
+    expect(app.title).toEqual('AngularApp');
   });
-
+/*
   it('should render title', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.debugElement.nativeElement;
     expect(compiled.querySelector('.content span').textContent).toContain('angular-app app is running!');
-  });
+  });*/
```

Comment:
looks like we don't need this test at all, so please remove it :)

Reply:
Oops, removed :)